lib/spack/spack/schema/concretizer.py

```diff
@@ -61,6 +61,7 @@
             "unify": {
                 "oneOf": [{"type": "boolean"}, {"type": "string", "enum": ["when_possible"]}]
             },
+            "compiler_mixing": {"type": "boolean"},
```

Comment:
Can you add `"description": "..."`? It's not common right now, but I'd like to add it since it shows up in editors during autocomplete and when hovering `compiler_mixing: ...`.